src/ansys/fluent/core/solver/flunits.py

```diff
@@ -104,119 +104,7 @@
-    "gas-constant": "J kg^-1 K^-1",
-    "heat-flux": "W m^-2",
-    "heat-flux-resolved": "m K s^-1",
-    "heat-generation-rate": "W m^-3",
-    "heat-transfer-coefficient": "W m^-2 K^-1",
-    "ignition-energy": "J mol^-1",
-    "kinematic-viscosity": "m^2 s^-1",
-    "length": "m",
-    "length-inverse": "m^-1",
-    "length-time-inverse": "m^-1 s^-1",
-    "mag-permeability": "H m^-1",
-    "mass": "kg",
-    "mass-diffusivity": "m^2 s^-1",
-    "mass-flow": "kg s^-1",
-    "mass-flow-per-depth": "kg m^-1 s^-1",
-    "mass-flow-per-time": "kg s^-2",
-    "mass-flux": "kg m^-2 s^-1",
-    "mass-transfer-rate": "kg m^-3 s^-1",
-    "moment": "N m",
-    "moment-of-inertia": "kg m^2",
-    "nucleation-rate": "m^-3 s^-1",
-    "number-density": "m^-3",
-    "percentage": "",
-    "power": "W",
-    "power-per-time": "W s^-1",
-    "pressure": "Pa",
-    "pressure-2nd-time-derivative": "Pa s^-2",
-    "pressure-gradient": "Pa m^-1",
-    "pressure-time-deriv-sqr": "Pa^2 s^-2",
-    "pressure-time-derivative": "Pa s^-1",
-    "resistance": "m^-1",
-    "soot-formation-constant-unit": "kg N^-1 m^-1 s^-1",
-    "soot-linear-termination": "m^3 s^-1",
-    "source-elliptic-relaxation-function": "kg m^-3 s^-2",
-    "source-energy": "W m^-3",
-    "source-kinetic-energy": "kg m^-1 s^-3",
-    "source-mass": "kg m^-3 s^-1",
-    "source-momentum": "N m^-3",
-    "source-specific-dissipation-rate": "kg m^-3 s^-2",
-    "source-temperature-variance": "K^2 m^-3 s^-1",
-    "source-turbulent-dissipation-rate": "kg m^-1 s^-4",
-    "source-turbulent-viscosity": "kg m^-1 s^-2",
-    "specific-area": "m^2 kg^-1",
-    "specific-energy": "J kg^-1",
-    "specific-heat": "J kg^-1 K^-1",
-    "spring-constant": "N m^-1",
-    "spring-constant-angular": "N m radian^-1",
-    "stefan-boltzmann-constant": "W m^-2 K^-4",
-    "surface-density": "kg m^-2",
-    "surface-tension": "N m^-1",
-    "surface-tension-gradient": "N m^-1 K^-1",
-    "temperature": "K",
-    "temperature-difference": "K",
-    "temperature-gradient": "K m^-1",
-    "temperature-inverse": "K^-1",
-    "temperature-variance": "K^2",
-    "thermal-conductivity": "W m^-1 K^-1",
-    "thermal-resistance": "m^2 K W^-1",
-    "thermal-resistivity": "m K W^-1",
-    "thermophoretic-diffusivity": "kg m^2 s^-2",
-    "time": "s",
-    "time-inverse": "s^-1",
-    "time-inverse-cubed": "s^-3",
-    "time-inverse-squared": "s^-2",
-    "turb-kinetic-energy-production": "kg m^-1 s^-3",
-    "turbulent-energy-diss-rate": "m^2 s^-3",
-    "turbulent-energy-diss-rate-gradient": "m s^-3",
-    "turbulent-kinetic-energy": "m^2 s^-2",
-    "turbulent-kinetic-energy-gradient": "m s^-2",
-    "velocity": "m s^-1",
-    "viscosity": "kg m^-1 s^-1",
-    "voltage": "V",
-    "volume": "m^3",
-    "volume-flow-rate": "m^3 s^-1",
-    "volume-flow-rate-per-depth": "m^3 s^-1 m^-1",
-    "volume-inverse": "m^-3",
-    "youngs-modulus": "N m^-2",
-}
+from ansys.fluent.core.constants import QuantityT, _fl_unit_table
```

Comment:
@hpohekar I'm not sure why moving these tables to a common `constants` module is needed(?)  cc: @prmukherj